packages/reactotron-app/App/Stores/UiStore.js

```diff
@@ -135,6 +136,11 @@ class UI {
     }
   }
 
+  @action
+  toggleTimelineOrder = () => {
+    this.isTimelineOrderReversed = !this.isTimelineOrderReversed
```

Comment:
It might be worth saving this so that you don't have to set it every time you open the app. Filters do something similar. I believe they store it into local storage.

Reply:
Done :)